Formula/nanopolish.rb

```diff
@@ -2,30 +2,18 @@ class Nanopolish < Formula
   # cite Loman_2015: "https://doi.org/10.1038/nmeth.3444"
   desc "Signal-level algorithms for MinION data"
   homepage "https://github.com/jts/nanopolish"
-  url "https://github.com/jts/nanopolish.git",
-        :tag => "v0.9.0",
-        :revision => "1c6ae110a1b7d0ca5072025b1889997fec0828ed"
-  head "https://github.com/jts/nanopolish.git"
-
-  bottle do
-    root_url "https://linuxbrew.bintray.com/bottles-bio"
-    sha256 "fc037c77203e00cb96937cea83aa524c97af083ec051a3c7a28cc6bb666b7415" => :sierra_or_later
-    sha256 "27bc64fe7504ef33c2463c41f7ee11807193368c4ac68d6f6575b037c3e83575" => :x86_64_linux
-  end
+  url "https://github.com/jts/nanopolish/archive/v0.10.1.tar.gz"
+  sha256 "ae31f83bfb3ad461265bedd315c31fe83e63a5509269d8a9e892a6813bcd9871"
```

Comment:
Please don't delete the `bottle` block and the `head` line. You can leave them as-is.

You can't unfortunately use the tarball archive, because it doesn't include some of the necessary git submodule dependencies. Use the git tag as above.
```ruby
        :tag => "v0.10.1",
        :revision => "0438b85dd3acf84a34c8e3d2098d55ab040900e3"
```
See https://github.com/jts/nanopolish/releases to get the SHA-1 (revision) for a given tag.